android/app/src/main/java/com/swmansion/gesturehandler/react/RNZoomableButtonManager.java

```diff
@@ -89,7 +94,9 @@ public void run() {
                         mIsTaskScheduled = false;
                         mIsLongTaskScheduled = true;
                         onReceivePressEvent(true);
-                        animate(false);
+                        if (!shouldLongPressEndPress) {
```

Comment:
I could remove this condition and make the pressed animation active while the button is being pressed, but it seems we want to have it scale back after some time

Reply:
this seems counterintuitive to me but @christianbaroni call, non blocking regardless